### PR TITLE
Rename Sticky Activities sample

### PR DESCRIPTION
**Pull request by lorensr**
To match https://github.com/temporalio/samples-typescript/pull/268

### CODE REVIEW

File: worker_specific_task_queues/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Worker-Specific Task Queues
+
+Use a unique Task Queue for each Worker in order to have certain Activities run on a specific Worker. In the Go and Java SDKs, this is explicitly supported via the Session option, but in other SDKs a different approach is required.
```
Comment (cretz) [inline worker_specific_task_queues/README.md:3]: There are no sessions in Java. I should have caught this when I reviewed the user's PR.

File: README.md [hunk 2]
```diff
@@ -51,7 +51,6 @@ Some examples require extra dependencies. See each sample's directory for specif
     while running.
   * [hello_signal](hello/hello_signal.py) - Send signals to a workflow.
 <!-- Keep this list in alphabetical order -->
-* [activity_sticky_queue](activity_sticky_queue) - Uses unique task queues to ensure activities run on specific workers.
```
Comment (cretz) [inline README.md:54]: Ug this link never worked. I should have noticed this on the user's original PR.

**Review by cretz — APPROVED** [review]
Couple of minor comments, up to you if you want to fix now or not

File: worker_specific_task_queues/README.md (hunk 1 above)
Comment (lorensr) [inline worker_specific_task_queues/README.md:3]:
```suggestion
Use a unique Task Queue for each Worker in order to have certain Activities run on a specific Worker. In the Go SDK, this is explicitly supported via the Session option, but in other SDKs a different approach is required.
```

**Comment by cretz** [comment]
LGTM. You may also want to do this to https://github.com/temporalio/samples-dotnet/tree/main/src/ActivityStickyQueues and https://github.com/temporalio/samples-go/tree/main/activities-sticky-queues or at least open the issues.

You can feel free to merge this or I can.

**Comment by cretz** [comment]
@lorensr - Just merged #78 which conflicts here. Feel free to fix and merge.

**Pull request merged**